addons/l10n_in_edi_ewaybill/models/account_edi_format.py

```diff
@@ -540,6 +540,12 @@ def _l10n_in_edi_ewaybill_check_authentication(self, company):
                 return True
         return False
 
+    def _l10n_in_get_error_message(self, response):
```

Comment:
@zepa-odoo Looking at it more closely, you are actually setting the error message in the response
```suggestion
    def _l10n_in_set_missing_error_message(self, response):
```

Reply:
Done.